src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -514,6 +516,28 @@ static int M()
         }
 
         [Fact]
+        public void DefaultInEnum()
+        {
+            string source = @"
+enum E
+{
+    DefaultEntry = default,
+    OneEntry = default + 1
```

Comment:
I'm going to guess that overload resolution finds a most specific overload of `operator +` that is applicable, and the result is a constant.